word_nerd/users/views.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.generics import CreateAPIView
+
+from .serializers import RegisterSerializer
+
+
+class RegisterUserApi(CreateAPIView):
```

Comment:
I don't know what naming convention django-rest uses, maybe we can use the same, e.g. `RegisterUserAPIView`?